static/app/components/feedback/feedbackItem/tagsSection.tsx

```diff
@@ -1,16 +1,25 @@
 import {KeyValueTable, KeyValueTableRow} from 'sentry/components/keyValueTable';
 import TextOverflow from 'sentry/components/textOverflow';
 import {Tooltip} from 'sentry/components/tooltip';
+import {space} from 'sentry/styles/space';
 
 interface Props {
   tags: Record<string, string>;
+  collapsed?: boolean;
 }
 
-export default function TagsSection({tags}: Props) {
+export default function TagsSection({tags, collapsed}: Props) {
   const entries = Object.entries(tags);
 
   return (
-    <KeyValueTable noMargin>
+    <KeyValueTable
+      noMargin
+      style={
+        collapsed
+          ? {maxHeight: '90px', overflow: 'hidden', marginBottom: `${space(1)}`}
```

Comment:
inline styles 

![gif](https://media.tenor.com/xy6-y9tLf0YAAAAM/homer-simpson-the-simpsons.gif)

Reply:
![image](https://media4.giphy.com/media/H5C8CevNMbpBqNqFjl/giphy.gif)